pkg/feature/templates/servicemesh/authorino/auth-smm.tmpl

```diff
@@ -0,0 +1,10 @@
+apiVersion: maistra.io/v1
+kind: ServiceMeshMember
+metadata:
+  name: default
```

Comment:
so you have one "default" in knative-serving NS , 
another "default" in $ApplicationName-auth-provider NS

Reply:
Yes. These are to enroll the namespaces to the Service Mesh. I think the name is not relevant.